pkg/apis/eventsource/v1alpha1/types.go

```diff
@@ -461,28 +461,31 @@ type KafkaEventSource struct {
 	Topic string `json:"topic" protobuf:"bytes,3,opt,name=topic"`
 	// Backoff holds parameters applied to connection.
 	ConnectionBackoff *apicommon.Backoff `json:"connectionBackoff,omitempty" protobuf:"bytes,4,opt,name=connectionBackoff"`
+	// SASL configuration for the kafka client
```

Comment:
BTW - we should keep the order of existing fields, otherwise it's a breaking change for proto files, we should reduce that kind of changes.

https://developers.google.com/protocol-buffers/docs/pythontutorial#extending-a-protocol-buffer

Reply:
Reverted the byte ordering and added the SASL configs to the end.